staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/testdata/valid-configs/aes-cbc-first.yaml

```diff
@@ -14,6 +14,11 @@ resources:
           name: testprovider
           endpoint: unix:///tmp/testprovider.sock
           cachesize: 10
+      - kms:
+          apiVersion: v2
+          name: testproviderv2
+          endpoint: unix:///tmp/testprovider.sock
```

Comment:
in the real world case, wouldn't v1 and v2 would have different endpoints?

Reply:
that's right, each kms provider in the list irrespective of the API version would have a unique endpoint if the goal is to have multiple providers configured for read